src/vm/eepolicy.cpp

```diff
@@ -1176,18 +1176,31 @@ inline void LogCallstackForLogWorker()
 // Return Value:
 //    None
 //
-inline void DoLogForFailFastException(LPCWSTR pszMessage, PEXCEPTION_POINTERS pExceptionInfo)
+inline void DoLogForFailFastException(LPCWSTR pszMessage, PEXCEPTION_POINTERS pExceptionInfo, LPCWSTR errorSource)
 {
     WRAPPER_NO_CONTRACT;
 
     Thread *pThread = GetThread();
     EX_TRY
     {
-        PrintToStdErrA("FailFast: ");
+        if (errorSource == NULL)
+        {
+            PrintToStdErrA("FailFast: ");
```

Comment:
The failfast message will now be printed in the next line. Intentionally? If yes, you could deliver the white space character

Reply:
Remove*